crates/services/gas_price_service/src/v1/metadata.rs

```diff
@@ -68,19 +67,56 @@ impl V1Metadata {
 }
 
 pub struct V1AlgorithmConfig {
-    new_exec_gas_price: u64,
-    min_exec_gas_price: u64,
-    exec_gas_price_change_percent: u16,
-    l2_block_fullness_threshold_percent: u8,
-    gas_price_factor: NonZeroU64,
-    min_da_gas_price: u64,
-    max_da_gas_price_change_percent: u16,
-    da_p_component: i64,
-    da_d_component: i64,
-    normal_range_size: u16,
-    capped_range_size: u16,
-    decrease_range_size: u16,
-    block_activity_threshold: u8,
+    pub new_exec_gas_price: u64,
+    pub min_exec_gas_price: u64,
+    pub exec_gas_price_change_percent: u16,
+    pub l2_block_fullness_threshold_percent: u8,
+    pub gas_price_factor: NonZeroU64,
+    pub min_da_gas_price: u64,
+    pub max_da_gas_price_change_percent: u16,
+    pub da_p_component: i64,
+    pub da_d_component: i64,
+    pub normal_range_size: u16,
+    pub capped_range_size: u16,
+    pub decrease_range_size: u16,
+    pub block_activity_threshold: u8,
+    pub unrecorded_blocks: Vec<(u32, u64)>,
```

Comment:
Hmm. I guess it doesn't hurt. I don't know why we'd need this if it wasn't already in the metadata.

Reply:
need this to bootstrap the `AlgorithmV1` with a state that we can test only DA data being sent :)

Reply:
Hmm. K. Technically you could provide a metadata with it, but that's expensive.